conftest.py

```diff
@@ -0,0 +1,10 @@
+import os
+import shutil
+
+os.environ["ALGOBOT_TESTING"] = "1"
+
+
+def pytest_unconfigure(config):
+    from algobot.helpers import PATHS, AppDirTemp
```

Comment:
Super important this is done here otherwise we init PATHS before setting the env var to control which one gets init